conf/node-default.properties

```diff
@@ -152,27 +152,23 @@
 
 ## Size of the download cache for blocks
 
-# brs.blockCacheMB = 40
+# node.blockCacheMB = 40
 
 ## Add this to check the deadline of every block since Genesis, otherwise only past the checkpoint.
 
-# brs.checkPointHeight = -1
+# node.checkPointHeight = -1
 
 #### API SERVER ####
 
 ## Accept http/json API requests.
 
 # API.Server = on
 
-## Accept gRPC API requests.
-
-# API.V2.Server = off
-
 ## JETTY pass-through options. See documentation at
 ## https://www.eclipse.org/jetty/documentation/9.2.22.v20170531/dos-filter.html
 ## API section:
 
-# JETTY.API.DoSFilter                   = off
+# JETTY.API.DoSFilter                   = on
```

Comment:
just asking: the Filters are disabled by default?

Reply:
they are now enabled by default and localhost is whitelisted